### PR TITLE
Include e10s status in reports

### LINKED ISSUE #25: Add in e10s status of Firefox
Need to pass through to the AMO API the e10s status of Firefox. We can find this by checking:

https://dxr.mozilla.org/mozilla-central/source/xpcom/system/nsIXULRuntime.idl#89
appInfo.browserTabsRemoteAutostart

Supports mozilla/addons#163

### PR DESCRIPTION
**Pull request by wagnerand**
Fixes #25 and #26.

### CODE REVIEW

File: lib/acr-util.js [hunk 1]
```diff
@@ -19,32 +19,23 @@ exports.Version = Version;
...
     var osVersion;
-    
+
     try
     {
         osVersion = this.getMostRecentAppWindow().navigator.oscpu;
     }
     catch (e) { Logger.warn(e); }
 
-    var info = Cc["@mozilla.org/xre/app-info;1"] .getService(Ci.nsIXULAppInfo);
```
Comment (tofumatt) [inline lib/acr-util.js]: Heh, interesting that ever worked.

File: lib/acr-api.js [hunk 2]
```diff
@@ -8,10 +8,10 @@ const Request = require("sdk/request").Request;
 const Logger = require("./acr-logger");
 const Util = require("./acr-util");
 
-const SERVICE_ROOT_URL = "https://%%AMO_HOST%%/en-US/firefox";
+const SERVICE_ROOT_URL = "http://%%AMO_HOST%%/en-US/firefox";
```
Comment (tofumatt) [inline lib/acr-api.js]: Why change this to `http`?

**Comment by tofumatt** [comment]
r+wc

This looks good, does it need anything to land in addons-server first?

File: lib/acr-util.js (hunk 1 above)
Comment (wagnerand) [inline lib/acr-util.js]: This is only the tip of the iceberg in terms of cleanup :)

File: lib/acr-api.js (hunk 2 above)
Comment (wagnerand) [inline lib/acr-api.js]: Oh, I meant to revert this before committing. Good catch!

**Comment by wagnerand** [comment]
Ideally yes. Although it won't affect current users until we push a new version of ACR to AMO.

**Comment by tofumatt** [comment]
r+

**Comment by wagnerand** [comment]
Thank you!

**Pull request merged**